strategy.js

```diff
@@ -89,8 +98,8 @@ Strategy.prototype.userProfile = function(accessToken, done) {
       profile.id = json.id;
       profile.displayName = json.displayName;
       profile.emails = json.emails;
-      profile.avatar = json.avatar;
-      profile.created = json.created;
+      //profile.avatar = json.avatar;
+      //profile.created = json.created;

```

Comment:
Go ahead and leave these two items here in case other people need them.  The intention is to publish this back to npm.

When we consume it, we just won't use those fields when they come back.

Reply:
done